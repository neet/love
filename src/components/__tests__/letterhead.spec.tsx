import * as React from 'react';
import * as ReactTestRenderer from 'react-test-renderer';
import Letterhead from '../letterhead';

describe('Letterhead', () => {
  it('rends a letterhead with a given react node', () => {
    const tree = ReactTestRenderer.create(
      <Letterhead>
        <p>This is a node</p>
      </Letterhead>,
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
