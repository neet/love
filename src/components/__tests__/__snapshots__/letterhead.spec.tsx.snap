// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Letterhead rends a letterhead with a given react node 1`] = `
<aside
  className="letterhead"
>
  <p>
    This is a node
  </p>
</aside>
`;
